packages/kbn-pm/src/cli.ts

```diff
@@ -23,6 +23,8 @@ function help() {
 
     Global options:
 
+       -e, --exclude        Exclude specified projects.
+       -i, --include        Include only specified projects. If left unspecified, it defaults to including all projects.
        --skip-kibana        Do not include the root Kibana project when running command.
```

Comment:
We can do it in a follow-up, but feels like we can get rid of `--skip-kibana` now 🎉

Reply:
Yep, I'll prepare a follow-up with updated `README.md`.